src/common/utils/kaltura-params.js

```diff
@@ -133,13 +158,28 @@ function addKalturaParams(player: Player, playerConfig: PartialKPOptionsObject):
     if (sources[key]) {
       sources[key].forEach(source => {
         if (typeof source.url === 'string' && source.url.toLowerCase().indexOf(PLAY_MANIFEST) > -1 && !source.localSource) {
-          updateSessionIdInUrl(source, playerConfig.session && playerConfig.session.id);
-          addReferrer(source);
-          addClientTag(source);
+          source.url = updateSessionIdInUrl(source.url, playerConfig.session && playerConfig.session.id);
+          source.url = addReferrer(source.url);
+          source.url = addClientTag(source.url);
+        }
+        if (source.drmData && source.drmData.length) {
+          source.drmData.forEach(drmData => {
+            if (
+              typeof drmData.licenseUrl === 'string' &&
+              drmData.licenseUrl.indexOf(UDRM_DOMAIN) > -1 &&
+              drmData.licenseUrl.indexOf(CUSTOM_DATA) > -1 &&
+              drmData.licenseUrl.indexOf(SIGNATURE) > -1
+            ) {
+              drmData.licenseUrl = updateSessionIdInUrl(drmData.licenseUrl, playerConfig.session && playerConfig.session.id, DRM_SESSION_ID);
```

Comment:
we use `playerConfig.session && playerConfig.session.id` twice now so take it outside and above(line `sources[key].forEach(source => {`) and then reference it in both the source and drm urls call to `updateSessionIdInUrl`

Reply:
done